proposals/0005-static-content-hash.md

```
@@ -0,0 +1,175 @@
+        options { call, outgoingContent ->
+            if (HttpHeaders.CacheControl in outgoingContent.headers) return@options null
+            when (outgoingContent.contentType?.withoutParameters()) {
+                ContentType.Text.JavaScript, ContentType.Text.CSS -> CachingOptions(CacheControl.MaxAge(maxAgeSeconds = 24 * 60 * 60))
+                else -> null
+            }
+        }
+}
+```
+
+# Advantages
+[advantages]: #advantages
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "advantages"

(MD053, link-image-reference-definitions)